azure-iot-provisioning-devicesdk/samples/register_symmetric_key.py

```diff
@@ -0,0 +1,52 @@
+import os
+import logging
+from azure.iot.provisioning.devicessdk.security.sk_security_client import SymmetricKeySecurityClient
```

Comment:
>devicessdk [](start = 28, length = 10)

this namespace has not been updated

Reply:
what kind of namespace updates are you referring ? are you referring to the ones that happened as part of discussion ?

---
In reply to: [270209007](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r270209007) [](ancestors = 270209007)